src/views/catalog/EnableInstanceTypeTechPreviewModal/EnableInstanceTypeTechPreviewModal.tsx

```diff
@@ -0,0 +1,149 @@
+import { Trans } from 'react-i18next';
+import instanceTypeTechPreviewModalImage from 'images/instance-type-tech-preview-modal-img.svg';
+
+import { useIsAdmin } from '@kubevirt-utils/hooks/useIsAdmin';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { usePreviewFeatures } from '@kubevirt-utils/hooks/usePreviewFeatures/usePreviewFeatures';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import {
+  Button,
+  ButtonVariant,
+  Modal,
+  ModalVariant,
+  Split,
+  SplitItem,
+  Stack,
+  StackItem,
+  Text,
+  TextVariants,
+  Title,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+type EnableInstanceTypeTechPreviewModalProps = {
+  navigateToCatalog: () => void;
+};
+
+const EnableInstanceTypeTechPreviewModal: FC<EnableInstanceTypeTechPreviewModalProps> = ({
+  navigateToCatalog,
+}) => {
+  const { t } = useKubevirtTranslation();
+
+  const { instanceTypesEnabled, loading, toggleInstanceTypesFeature, error } = usePreviewFeatures();
+
+  const isAdmin = useIsAdmin();
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  useEffect(() => {
+    if (!isEmpty(error)) {
+      setIsOpen(true);
+      return;
+    }
+    if (!loading) {
+      setIsOpen(!instanceTypesEnabled);
+    }
+  }, [loading, instanceTypesEnabled, error]);
+
+  const onClose = useCallback(() => {
+    setIsOpen(false);
+    navigateToCatalog();
+  }, [navigateToCatalog]);
+
+  return (
+    <Modal
+      header={
+        <Title headingLevel="h1">{t('Creating a VirtualMachine from an InstanceType')}</Title>
+      }
+      isOpen={isOpen}
+      onClose={onClose}
+      variant={ModalVariant.medium}
+    >
+      <Trans t={t} ns="plugin__kubevirt-plugin">
+        <Stack hasGutter>
+          <StackItem>
+            <Text component={TextVariants.p} className="pf-u-font-size-xl pf-u-danger-color-100">
+              Creating VirtualMachines is easier than ever!
+            </Text>
+          </StackItem>
+          <StackItem>
+            <Text component={TextVariants.p}>
+              This{' '}
+              <Text
+                component={TextVariants.a}
+                href="https://access.redhat.com/support/offerings/techpreview"
+                target="_blank"
+              >
+                Developer Preview feature
+              </Text>{' '}
+              provides a simple and quick way to create VirtualMachines.{' '}
+              <Text component={TextVariants.p}>
+                You must have cluster admin permissions to enable it. You can disable this feature
+                by going
+              </Text>{' '}
+              <Text component={TextVariants.p}>to the settings tab in the overview page.</Text>
+            </Text>
+          </StackItem>
+        </Stack>
+      </Trans>
+      <br />
+      <Split>
+        <SplitItem>
+          <Stack hasGutter>
+            <StackItem>
```

Comment:
I'd better avoid using `Split` and `Stack` components like this. I think it is unnecessary to achieve the expected look.

When I was implementing Welcome modal, I used `Grid` component to divide the space into 2 columns. In your case, the 2 links could go to the 1st column, and the image to the second one. Simple as that. Maybe I also needed to use `Stack` component to get better look, but only one time. The result was much simple, more readable code.

With this approach you could also avoid using `<br />` a few lines above. Using `Grid` + eventually some css if needed would do the thing. You don't have to do it, take it just as my suggestion ;) Thanks.

Reply:
the welcome modal is looking a bit different than this one by design, I tried to use grid but it wasn't fitting as I wanted and needed to make more css modifications that it's not worth it. about readability, I can make this component a bit easier on the eyes with extracting components to separated files